### DISCUSSION TITLE
Why am I *required* to have some form of clipboard manager?

### DISCUSSION THREAD

**Discussion by raekuul** [lxqt/lxqt#1938] [Q&A]
Debian 11 "Bullseye" (Testing)
I'm looking at the dependencies for lxqt and it says I need qlipper or clipit or xfce4-clipman.

Why do I *need to have* a clipboard manager at all? Can't this be optional?

If this is a debian-specific requirement, please let me know so I can contact the correct people.

**Comment by stefonarch** [comment]
Sure this is a decision from debian maintainers, there is an email:
```
$ apt show lxqt
Package: lxqt
Version: 30
Priority: optional
Section: metapackages
Source: lxqt-metapackages
Maintainer: LXQt Packaging Team <pkg-lxqt-devel@lists.alioth.debian.org>
```
I think most distros ship qlipper together with their "full" LXQt, if you want only the essential LXQt install` lxqt-core` package.
```

$ apt depends lxqt-core
....
 |Recommends: qlipper
...
```
